File: examples/node/address-overview.mjs
import { OneDexClient } from '../../packages/js/src/index.js';

const client = new OneDexClient({
  baseUrl: process.env.ONEDEX_BASE_URL,
});

const response = await client.overview.address({
  address: process.argv.slice(2).join(' ') || '10 rue des cordeliers aix',
  dvf_radius_m: 600,
});

console.log(JSON.stringify(response, null, 2));

File: examples/python/address_overview.py
import json
import os
import sys
from pathlib import Path

sys.path.insert(0, str(Path(__file__).resolve().parents[2] / "packages/python/src"))

from onedex import OneDexClient


client = OneDexClient(
    base_url=os.environ.get("ONEDEX_BASE_URL"),
)

response = client.overview.address({
    "address": " ".join(sys.argv[1:]) or "10 rue des cordeliers aix",
    "dvf_radius_m": 600,
})
print(json.dumps(response, indent=2, ensure_ascii=False))
